Update for archiving

geeknoid opened the pull request.

ldemailly commented on _includes/archives.html:
```diff
@@ -0,0 +1,19 @@
+
+                var li = e.appendChild(document.createElement("li"));
+                var a = li.appendChild(document.createElement("a"));
+                a.setAttribute("href", "https://archive.istio.io/" + lines[i]);
```
needs the trailing "/" here

ldemailly commented on _includes/nav.html:
```diff
@@ -24,7 +24,11 @@
                 <img src="{{home}}/img/logo.png" alt="Istio" width="36px" height="54px"/>
-                <span class="brand-name">Istio</span>
+                {% if site.data.istio.archive %}
+                  <span class="brand-name">Istioldie {{site.data.istio.short_version}}</span>
```
`Istioldie` was fine/fun for the top level of https://archive.istio.io/ but can we just have
`Istio archive v{{site.data.istio.short_version}}` here
  
Also I made it smaller than Istio in https://archive.istio.io/v0.2/docs/ for instance, which I think is useful
(add another span/class for the archive tag)

ldemailly commented on _layouts/base.html:
```diff
@@ -21,13 +21,21 @@
-    <title>Istio</title>
+    {% if site.data.istio.archive %}
+        {% if page.title == 'Istio' %}
+            <title>Istioldie {{site.data.istio.short_version}}</title>
```
ditto

ldemailly commented on _sass/base/_archive_colors.scss:
```diff
@@ -0,0 +1,6 @@
+// Colors for the archive version of the site
+$mainBrandColor: #466BB0;
+$secondBrandColor: #CEAF0C;
```
let's not change the colors it's confusing for users (and not good for the brand)
as long as each page has that "archive" marking it's good

geeknoid commented:
I fixed the missing /

I also removed the need to run JavaScript to display the dropdown. As a result, I'm happy to put the dropdown on every page, so I removed the special "site archives" page.

I'd like to keep the visuals as-is and we'll see what others think. Easily changed later.

ldemailly commented:
can you add a link or screenshots so "others" can decide on the look & feel

jasminejaksic commented:
+100 to no "Istioldies" and gradient (leave alone blue to yellow) - it reminds me of the 90s Flash-based-websites :)

geeknoid commented:
Refreshed colors for archived sites: gold be gone.

Fixed search in archived sites.

ldemailly commented:
thanks, oldie is still there
